test/src/e2e_vm_tests/test_programs/should_pass/language/nested_while_and_if/src/main.sw

```diff
@@ -1,6 +1,6 @@
 script;
 
-use core::*;
+use std::*;
```

Comment:
This line of code doesn't actually do anything, so we should be able to remove it.